.github/workflows/on-demand-vrfv2plus-performance-test.yml

```diff
@@ -25,79 +25,63 @@ on:
-          echo "### Test secrets override key" >> $GITHUB_STEP_SUMMARY
-          echo "${{ inputs.test_secrets_override_key }}" >> $GITHUB_STEP_SUMMARY           
-      - name: Show chainlink version in summary
-        run: |
-          echo "### Chainlink version" >> $GITHUB_STEP_SUMMARY
-          echo "${{ inputs.chainlink_version }}" >> $GITHUB_STEP_SUMMARY                  
-      - name: Run Tests
-        uses: smartcontractkit/chainlink-github-actions/chainlink-testing-framework/run-tests@94cb11f4bd545607a2f221c6685052b3abee723d # v2.3.32
-        env:
-          E2E_TEST_CHAINLINK_IMAGE: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ secrets.QA_AWS_REGION }}.amazonaws.com/chainlink
-          E2E_TEST_CHAINLINK_VERSION: ${{ inputs.chainlink_version }} # Default chainlink version if not overrided in the test config 
-          E2E_TEST_LOKI_TENANT_ID: ${{ secrets.GRAFANA_INTERNAL_TENANT_ID }}
-          E2E_TEST_LOKI_ENDPOINT: https://${{ secrets.GRAFANA_INTERNAL_HOST }}/loki/api/v1/push
-          E2E_TEST_LOKI_BASIC_AUTH: ${{ secrets.GRAFANA_INTERNAL_BASIC_AUTH }}
-          E2E_TEST_GRAFANA_BASE_URL: "http://localhost:8080/primary"
-          E2E_TEST_GRAFANA_DASHBOARD_URL: "/d/ddf75041-1e39-42af-aa46-361fe4c36e9e/ci-e2e-tests-logs"
-          E2E_TEST_GRAFANA_BEARER_TOKEN: ${{ secrets.GRAFANA_INTERNAL_URL_SHORTENER_TOKEN }}
-        with:
-          test_command_to_run: cd ./integration-tests/load && go test -v -count=1 -timeout 24h -run "${{ inputs.test_list_regex }}" ./vrfv2plus
-          test_download_vendor_packages_command: cd ./integration-tests && go mod download
-          test_secrets_override_base64: ${{ secrets[inputs.test_secrets_override_key] }}
-          test_config_override_path: ${{ inputs.test_config_override_path }}
-          aws_registries: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}
-          artifacts_name: vrf-test-logs
-          artifacts_location: ./integration-tests/load/vrfv2plus/logs/
-          token: ${{ secrets.GITHUB_TOKEN }}
-          go_mod_path: ./integration-tests/go.mod
-          should_cleanup: false
-          QA_AWS_REGION: ${{ secrets.QA_AWS_REGION }}
-          QA_AWS_ROLE_TO_ASSUME: ${{ secrets.QA_AWS_ROLE_TO_ASSUME }}
-          QA_KUBECONFIG: ${{ secrets.QA_KUBECONFIG }}
+          TEST_CMD='cd integration-tests/load && go test -v -count=1 -timeout 24h -run "${{ inputs.test_list_regex }}" ./vrfv2plus'
+          TEST_CONFIG_OVERRIDE_PATH=${{ inputs.test_config_override_path }}
+          TEST_TYPE=${{ inputs.performanceTestType }}
+
+          TEST_LIST=$(jq -n -c \
+            --arg test_cmd "$TEST_CMD" \
+            --arg test_config_override_path "$TEST_CONFIG_OVERRIDE_PATH" \
+            --arg TEST_TYPE "$TEST_TYPE" \
+            '{
+              "tests": [
+                {
+                  "id": "TestVRFv2Plus_Performance",
+                  "path": "integration-tests/load/vrfv2plus/vrfv2plus_test.go",
+                  "runs_on": "ubuntu22.04-8cores-32GB",
```

Comment:
Is this a performance test running directly in GHA? If so, I think we should change it to run in K8s. If not, this runner is way overkill.

Reply:
@kalverra this is the same runner as before. I don't want to change it in this PR unless @iljapavlovs agrees that we can run on cheaper runner

Reply:
yes, we can try using default runner

Reply:
ok, let me change it to `ubuntu-latest`. @iljapavlovs please change it back in the future if needed